web/modules/custom/dpl_base/dpl_base.install

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * Base update hooks.
+ *
+ * These get run BEFORE config-import.
+ * This is where you enable/disable modules, as core.extension.yml is in
+ * config_ignore as library administrators can add their own modules.
+ */
+
+/**
+ * Run on the initial site setup. Remember to add individual update hooks.
+ */
+function dpl_base_install(): string {
+  $messages[] = dpl_base_update_10001();
+
+  return implode('\r\n', $messages);
+}
```

Comment:
I do not understand how to interpret the code comment "Remember to add individual update hooks."

Do we need to add all future update hooks here as well?

Reply:
Normally we use update hooks in the profile for general site updates, right?